### PR TITLE
Add new svg icons

### PR DESCRIPTION
**Pull request by JasonLG1979**
This changes the Spotify flatpak to use a svg full color icon and adds a symbolic icon.

@TingPing you'll have to help me out. not sure if I'm doing this right as I don't know how to test it.

### CODE REVIEW

**Comment by JasonLG1979** [comment]
bot, build com.spotify.Client

**Comment by JasonLG1979** [comment]
Yep. Still have no idea what I'm doing. The icons are there I just have no idea how to add them to ```com.spotify.Client.json```.

**Comment by JasonLG1979** [comment]
@TingPing the build fails with ```install: cannot stat 'com.spotify.Client.svg': No such file or directory``` everytime. I'm going to stop wasting the bots time until you can have a look at what I got.

**Comment by JasonLG1979** [comment]
It builds successfully now but the icons aren't in the right place because they don't show up.

https://github.com/flathub/com.spotify.Client/pull/98/commits/720334282a976ead1b8bd18af545bece2961b0f7#diff-83fd27ba24db1f36e7de08ce56331feaR114-R115

Is not right.

File: com.spotify.Client.json [hunk 1]
```diff
@@ -111,6 +111,8 @@
                 "install get-scale-factor.py xsettings.py /app/bin",
                 "install spotify-bin /app/bin/spotify",
                 "install -Dm644 com.spotify.Client.appdata.xml /app/share/appdata/com.spotify.Client.appdata.xml",
+                "install -Dm644 com.spotify.Client.svg /export/share/icons/hicolor/scalable/apps/com.spotify.Client.svg",
```
Comment (TingPing) [inline com.spotify.Client.json]:
```suggestion
                "install -Dm644 com.spotify.Client.svg /app/share/icons/hicolor/scalable/apps/com.spotify.Client.svg",
```

File: com.spotify.Client.json [hunk 2]
```diff
@@ -111,6 +111,8 @@
...
+                "install -Dm644 com.spotify.Client-symbolic.svg /export/share/icons/hicolor/symbolic/apps/com.spotify.Client-symbolic.svg",
```
Comment (TingPing) [inline com.spotify.Client.json]:
```suggestion
                "install -Dm644 com.spotify.Client-symbolic.svg /app/share/icons/hicolor/symbolic/apps/com.spotify.Client-symbolic.svg",
```

**Comment by JasonLG1979** [comment]
Ok @TingPing that build seems to work. Do you want to change the icon ref in com.spotify.Client.appdata.xml to point to the new full color icon svg also?

**Comment by JasonLG1979** [comment]
Yep, works like a charm. (in Fedora 31 nightly on Gnome 3.34.1) The icon even shows up in my extension after I'm sure to look for the flatpak'd icon name. (Spotify still ofc sends "spotify" as it's desktop entry)

Screenshot from 2019-10-12 17-30-44

**Comment by TingPing** [comment]
> Do you want to change the icon ref in com.spotify.Client.appdata.xml to point to the new full color icon svg also?

Yes that would be good.

**Comment by JasonLG1979** [comment]
Can it the new url be?:
https://raw.githubusercontent.com/flathub/com.spotify.Client/master/com.spotify.Client.svg

Or does it need proper image hosting somewhere?

**Comment by JasonLG1979** [comment]
> Since it is installed locally you don't need the `remote` type. In theory you don't need to add anything but testing appreciated.

OK then the icon that's displayed in the app store comes from the flathub repo not the flatpak package?

**Comment by TingPing** [comment]
`appstream-compose` extracts icons referenced from the package into the repo basically.

**Comment by JasonLG1979** [comment]
> `appstream-compose` extracts icons referenced from the package into the repo basically.

Yep, I noticed Pithos does not have an icon reference in it's appdata.xml.

**Comment by TingPing** [comment]
For Pithos for example: appdata -> launchable tag -> desktop file -> `Icon`

**Pull request merged**